fix(curriculum): added missing html boilerplate

Jackkv opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or Gitpod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #60074

<!-- Feel free to add any additional description of changes below this line -->

Added the missing HTML boilerplate to ensure consistency with the rest of the labs.

**Update 1:** changed the boilerplate title as suggested [here](https://github.com/freeCodeCamp/freeCodeCamp/issues/60074#issuecomment-2840189735).
**Update 2:** changed the spacing as requested [here](https://github.com/freeCodeCamp/freeCodeCamp/pull/60072#pullrequestreview-2805178089).

Jackkv committed:
fix(curriculum): added missing html boilerplate

huyenltnguyen reviewed (commented):
Hey @Jackkv, thank you for the PR.

Could you help link the issue that this PR is addressing? If there isn't an issue open for this change, please help create an issue first.

jdwilkin4 reviewed (commented):
This will need to updated to matches the changes listed here

https://github.com/freeCodeCamp/freeCodeCamp/issues/60074#issuecomment-2840189735

ilenia-magoni commented:
please update your PR as indicated
also please keep in mind that there is no need to merge main into your PR unless you are asked to do that (or there is a merge conflict)

Jackkv committed:
changed boilerplate title as suggested

Jackkv commented:
> This will need to updated to matches the changes listed here
> 
> [#60074 (comment)](https://github.com/freeCodeCamp/freeCodeCamp/issues/60074#issuecomment-2840189735)

I changed the title as suggested, now it should be in line with the other labs.

> please update your PR as indicated also please keep in mind that there is no need to merge main into your PR unless you are asked to do that (or there is a merge conflict)

Sorry, i merged by mistake but i have reverted it now.

Thanks to both.

a2937 reviewed (changes requested):
Please update the spacing to match the provided boilerplate text as mentioned in https://github.com/freeCodeCamp/freeCodeCamp/issues/60074#issuecomment-2840189735

Jackkv committed:
changed spacing as requested

Jackkv commented:
> Please update the spacing to match the provided boilerplate text as mentioned in [#60074 (comment)](https://github.com/freeCodeCamp/freeCodeCamp/issues/60074#issuecomment-2840189735)

I've made the requested changes. I had some doubts, so I force-pushed to amend the text capitalization, i hope that's okay. Now it should be consistent with the other labs.
